WRITING_GUIDE.md

```diff
@@ -0,0 +1,410 @@
+document is a candidate for localization (translation).  Make sure the
+English content is correct and clear.  This will help translators
+produce better output for non-native English users.
+
+### Be concise
+
+In technical documentation, typically *less is more*.  Avoiding long,
+complex sentences that contain multiple phrases and clauses make the
+content easier for the users to read and understand.  The [United
+States Plain Lanuguage
+Guidelines](https://www.plainlanguage.gov/guidelines/concise/) has
+several tips and examples to empliment concise language.  These
+include:
+
+* check your prepositions
+* omit redundant words
+* cut excess modifiers
+* avoid doublets and triplets
+
+### Use short sentences
+
+Keep sentences at 20 words or fewer.  Longer sentences are more
+difficult for readers to comprehend.
+
+Vary the length of sentences.  This helps keep readers interested.
+This will also give you better control of your content's tone.  A
+paragraph with only short sentences can unintentionally sound terse.
+The occasional longer sentence adds a bit of narrative interest, and
+can help a piece of writing sound friendlier.
+
+### Keep the content conversational
+
+Clear and direct content will help the user understand the content.
+Keep the following items in mind while adding content.
+
+* Use active voice.  Active voice supports brevity and helps the
+  reader know who will perform the action.
+* Avoid overly technical terms and jargen.  Use simple languge for
+  technical items when possible.
+* Use the present tense.  This will make the writing simpler, more
+  direct, and more forceful.
+* Use contractions.  Write like you talk.  However, avoid unclear
+  contractions (for example, *there'd*, *it'll*, and *they'd*.)
+* Use examples, but not too much.  Examples help clarify concepts.
+  However, do not over use examples as they break up the text and can
+  make it more difficult for the reader to follow.
+* Avoid Latin abbreviations.  Use the English meaining instead.  That
+  is, use "for example" instead of "e.g." and "that is" instead of
+  "i.e.".  The abbreviation "etc." is acceptable.
+* Use "must" to indicate requirements.
+* Do not use slashes, unless when they are part of a standard
+  technical term, such as *TCP/IP* or *client/server*.
+
+### Avoid duplictaion
+
+If something is written once and links to relevant information easily
+and well, users are more likely to trust the content. Duplicate
+content produces poor search results and confuses the user.  It can
+also lead to conflicting informtion.
+
+Instead of reproducing in-depth instruction, link to reliable,
+authoritative web pages with relevant information.
+
+When instructing users to contact the help desk, link to the [Getting
+Help](source/help/index.rst) page with "Contact the X help desk."
+instead of listing the email address.
+
+
+## Documentation content
+
+When determining what to include in the RDHPCS documentation, consider
```

Comment:
Suggest "As you decide what to include" rather than "in determining what to include".